[now-node] Export * does not pull in files

Hi All,

I was using "@now/node@0.11.1", recently tried to update to 12 and I am now getting errors all over saying certain files can not be found. Looking into it more I noticed that any file i call using export does not  get moved into the compiled directory. For instance I have:

api
-hook
--index.js
component
-week.js
-index.js

index.js has:
export * from './week';

and api/hook/index.js has:

import {CurrentWeek} from '../../component';
...

when I try and run hook/index.js I get:
Cannot find module './week' and if I look into the folder component of the compiled code the file week.js has not been moved in.

--- a/packages/now-node/package.json
+++ b/packages/now-node/package.json
@@ -28,7 +28,7 @@
     "@types/etag": "1.8.0",
     "@types/test-listen": "1.1.0",
     "@zeit/ncc": "0.20.4",
-    "@zeit/node-file-trace": "0.2.4",
+    "@zeit/node-file-trace": "0.2.5",
     "content-type": "1.0.4",
     "cookie": "0.4.0",
     "etag": "1.8.1",

